bin/ci-publish.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -ex
+
+npmPublish() {
+  npm publish
+}
+
+currentVersion() {
+  grep version <package.json | sed 's/[^0-9.]*\([0-9.]*\).*/\1/'
+}
+
+currentProject() {
+  npm ls --depth=-1 | head -1 | cut -f 1 -d " " | sed 's/@.*//'
+}
+
+checkVersion() {
+  local gitTag
+  gitTag=$(git describe --tags)
+
+  if [ "$(currentVersion)" != "$gitTag" ]; then
```

Comment:
I don't think this will solve the problem that we ran into today. The bump script succeeded, so the version bump actually happened on git, but when CircleCI auto-started the master merge tests, they failed so it did not get published. With this check, it will still pass on the next pull request.